templates/anndata_conversion.ipynb

```diff
@@ -0,0 +1,784 @@
+{
```

Comment:
<p>We can move this section to the <code>data_types.md</code> file in the documentation. Specifically, explaining the columns, what's in <code>obs</code> vs <code>obsm</code>, etc. should be something the user should get a good understanding of before running this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1079/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>